### DISCUSSION TITLE
Mobile toolbar issue

### DISCUSSION THREAD

**Discussion by Bojji-git** [phibr0/obsidian-commander#108] [General]
Hi there, @phibr0

After the most recent Obsidian update for mobile (v1.4.16), my toolbar interface changed. I used to have my toolbar commands distributed in 2 rows, but now, though under Commander's Toolbar Row Count I still have it set to present in 2 rows, the toolbar won't wrap the icons to form another row, as seen below.
![IMG_20231016_082925.jpg](https://github.com/phibr0/obsidian-commander/assets/146337291/23bc75b3-2f1d-44ed-96fb-ad1416433e60)

Is there a way to fix this?

Thanks in advance.

**Comment by duch11** [comment]
I'm just here to be that guy saying: "I too experience this issue" 😅🔥

**Comment by witecat** [comment]
Same for me. Mobile toolbar"forgot" it's multi-row setting and is showing all icons on one long row with empty space under. Android 12 tablet.
Very annoying, sorting down my workflow considerably
